backend/src/controllers/review.js

```diff
@@ -68,7 +68,7 @@ exports.getReplyLikes = function ({ params }, res) {
 
 /* PUT updated reply likes value */
 exports.putReplyLikes = function ({ user, params, body, query }, res) {
-    body.userID = user || ANONYMOUS
+    body.userID = user && user.id || ANONYMOUS
```

Comment:
Just looking at this.. Is there ever a reason you need more than userID? If not simple to put a middleware to expose this to all controllers.

Reply:
Sidenote, are we maintaining support for anonymous? Also if we are, does anonymous have details associated with them.

Reply:
this is temporary until we're authenticating all put/post/delete routes.
anonymous won't be supported after that imo

Reply:
kk